analysis/perf_analyzer.go

```diff
@@ -1,20 +1,25 @@
 package analysis
 
 import (
+	"encoding/json"
+	"fmt"
 	"reflect"
+	"strings"
+	"sync"
 	"unsafe"
 
 	"github.com/sarchlab/akita/v3/sim"
 )
 
 // PerfAnalyzerEntry is a single entry in the performance database.
 type PerfAnalyzerEntry struct {
-	Start sim.VTimeInSec
-	End   sim.VTimeInSec
-	Where string
-	What  string
-	Value float64
-	Unit  string
+	Start  sim.VTimeInSec
+	End    sim.VTimeInSec
+	Src    string
```

Comment:
So this is something weird. Where and What are clear terms, but Src and Linker are not. This entry is design to describe scalar data that is recorded at a certain location. Now, we have some different requirements, and the original design does not work. Shall we create a new struct called `TrafficEntry` and keep this one as `PerfEntry`?